source/_static/style.css

```diff
@@ -0,0 +1,117 @@
+body {
+	background-image: url("../_static/img/sunpy-bg.jpg");
+	background-repeat: no-repeat;
+	background-position: center center;
+	background-attachment: fixed;
+	background-size: cover;
+	color: #212121;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-size: 16px;
+}
+@font-face {
+	font-family: 'Source Sans Pro', sans-serif;
+	src: url('fonts/SourceSansPro-Bold.ttf');
+}
+a {
+	color: #FF5722;
+}
+a:hover {
+	color: #DD2C00;
+}
+h1 {
+	margin-top: 45px;
+}
+h4 {
+	font-weight: 300;
+	padding: 15px 0px 15px 0px;
+	font-size: 21px;
+}
+p.content  {
+	line-height: 24px;
+}
+p.content.last {
+	margin-bottom: 100px;
+}
+p.content.noheader {
+	padding-top: 15px;
+}
+li.dropdown.global-container {
+	display: none !important;
+}
+/*Navbar*/
+.navbar {
+	background-color: #212121;
```

Comment:
I am not a massive fan of this dark blue background colour. Maybe black instead?